src/index.tsx

```diff
@@ -327,7 +327,15 @@ export function formAtom<Fields extends FormFields>(
     submitStatus: submitResultAtom,
     submitCount: submitCountAtom,
     reset: resetAtom,
-  });
+  };
+
+  if (process.env.NODE_ENV !== "production") {
```

Comment:
Think you'll want to do
```suggestion
  if (typeof process !== "undefined" && process.env.NODE_ENV !== "production") {
```
 To avoid breaking bundles without a process polyfill

Reply:
fixed https://github.com/jaredLunde/form-atoms/pull/43/commits/85e0272688d08f45a3d3635e8a7c052e75feced3